packages/block-library/src/navigation/edit.js

```diff
@@ -26,6 +27,7 @@ import {
 	Placeholder,
 	Spinner,
 	ToolbarGroup,
+	Toolbar,
```

Comment:
Nitpick: alphabetically, `Toolbar` should come before `ToolbarGroup`. The rest of the imports in this list are listed alphabetically.